connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -510,7 +510,13 @@ private void startGame() {
         startGamePacket.setItemEntries(ItemRegistry.ITEMS);
         startGamePacket.setVanillaVersion("*");
         // startGamePacket.setMovementServerAuthoritative(true);
-        upstream.sendPacket(startGamePacket);
+        upstream.sendPacketImmediately(startGamePacket);
```

Comment:
Instead of sleeping the thread, match with the rest of the Geyser codebase by using:

`connector.getGeneralThreadPool().schedule(() -> startGamePacket, 500, TimeUnit.MILLISECONDS)`

Reply:
As discussed this will not achieve the aim but it gave me the idea to utilize the Threadpool for the packets that come afterwards which I'm a happier with rather than tying up a thread.

Note that #656 will need to be modified for this to work so will need to be merged afterwards.